Backend/src/main/java/Digitact/Backend/Model/User/ApplicantUI.java

```diff
@@ -106,6 +110,14 @@ public Title getTitle() {
         return title;
     }
 
+    public String getAdditionalInfo() {
+        return additionalInfo;
+    }
+
+    public void setAdditionalInfo(String impression) {
+        this.additionalInfo = impression;
+    }
```

Comment:
It is not impression. it is just additional information that is being given applicant. Can you give different name?